data/websites.json

```diff
@@ -41,15 +41,20 @@
       "description": "Official Website of Bedkot Municipality"
     },
     {
-      "name": "Belauri Municipality",
-      "url": "https://belaurimun.gov.np/",
-      "description": "Official Website of Belauri Municipality"
+      "name": "Besishahar Municipality",
+      "url": "https://besishaharmun.gov.np/",
+      "description": "Official Website of Besishahar Municipality"
     },
     {
       "name": "Belbari Municipality",
       "url": "https://belbarimun.gov.np/",
       "description": "Official Website of Belbari Municipality"
     },
+    {
+      "name": "",
```

Comment:
You have left the duplicate portion exactly as it is.

Reply:
Sorry, my bad I forgot to remove the unnecessary file changes that I made previously. Should be good now. Thanks.